db/migrate/20200813105100_remove_foreign_keys.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveForeignKeys < ActiveRecord::Migration[5.2]
+  def change
+    remove_foreign_key :user_preferences, :school_groups
+    remove_foreign_key :vacancies, :school_groups
```

Comment:
What replaces these removed foreign keys?

Reply:
Nothing for now